float64[] pos_x
float64[] pos_y
float64[] pos_z
float64[] ori_x
float64[] ori_y
float64[] ori_z
float64[] ori_w
---
int32 followed_poses
